Crashlytics/Crashlytics/Components/FIRCLSContext.h

```diff
@@ -96,10 +97,12 @@ typedef struct {
 } FIRCLSContextInitData;
 
 #ifdef __OBJC__
-bool FIRCLSContextInitialize(FIRCLSInternalReport* report,
-                             FIRCLSSettings* settings,
-                             FIRCLSFileManager* fileManager);
-
+bool FIRCLSContextInitialize(FIRCLSContextInitData* initData, FIRCLSFileManager* fileManager);
```

Comment:
Do we still need to pass `fileManager` as another argument here? Seems init Data also has a reference of `fileManager `